README.md

```diff
@@ -940,6 +946,176 @@ class GetItemsForUserTests(TestCase):
         self.assertEqual(expected, result)
 ```
 
+## Celery
+
+We use [Celery](http://www.celeryproject.org/) for the following general cases:
+
+* Communicating with 3rd party services (sending emails, notifications, etc.)
+* Offloading heavier computational tasks outside the HTTP cycle.
+* Periodic tasks (using Celery beat)
+
+We try to treat Celery as if it's just another interface to our core logic - meaning - **don't put business logic there.**
+
+An exmaple task might look like this:
+
+```python
+from celery import shared_task
+
+from project.app.services import some_service_name as service
+
+@shared_task
+def some_service_name(*args, **kwargs):
+    service(*args, **kwargs)
+```
+This is a task, having the same name as a service, which holds the actual business logic.
+
+**Of course, we can have more complex cituations**, like a chain or chord of tasks, each of them doing different domain relateed logic. In that case, it's hard to isolate everything in a service, because we now have dependencies between the tasks.
```

Comment:
```suggestion
**Of course, we can have more complex situations**, like a chain or chord of tasks, each of them doing different domain related logic. In that case, it's hard to isolate everything in a service, because we now have dependencies between the tasks.
```